ec2/spark_ec2.py

```diff
@@ -70,7 +70,7 @@
     "1.2.1",
     "1.3.0",
     "1.3.1",
-    "1.4.0"
+    "1.4.0",
```

Comment:
I was just copying the convention of what was there before in the master branch - both arrays had trailing commas, so I figured it was on purpose.